pegasus/sites.v3/hourofcode.com/public/css/prizes-page.css

```diff
@@ -0,0 +1,20 @@
+.prizes-logo {
+  width: 120px;
+  height: 120px;
+  position: relative;
+  float: left;
+  margin-right: 20px;
+}
+
+.prizes-logo img {
+    max-height: 100%;
+    max-width: 100%;
+    width: auto;
+    height: auto;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin: auto;
+}
```

Comment:
Ah yes. Good catch. Too much copying and pasting code from the internet.

Although... since I'm moving the images into a haml view, I'm going to move these pieces of css into common.css and delete this file altogether.